### PR TITLE
Version 1.4.2-rc1 release

### PR DESCRIPTION
**Pull request by garg-mudit**
### Changed
- Adjusted the minimum required `firebase/php-jwt` package version to 6.0.

### CODE REVIEW

File: src/Configuration.php [hunk 1]
```diff
@@ -109,7 +111,7 @@ class Configuration
      *
      * @var string
      */
-    protected $userAgent = "Swagger-Codegen/5.1.0/php";
+    protected $userAgent = 'Swagger-Codegen/v2.1/1.4.2-rc1/php/' . PHP_VERSION;
```
Comment (shan285) [inline src/Configuration.php:114]: Has the version changed from 5.1.0 to 1.4.2-rc1?

Comment (shan285) [reply src/Configuration.php:114]: As discussed I see that the configuration was on swaggercodegen ignore. Hence this is not an issue

**Review by shan285 — APPROVED** [review]
LGTM

**Review by shan285 — APPROVED** [review]
LGTM

**Comment by garg-mudit** [comment]
Fixes [Issue#11](https://github.com/docusign/docusign-admin-php-client/issues/11)

**Pull request merged**